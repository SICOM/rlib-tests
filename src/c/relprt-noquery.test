COMMAND="./relprt --no-query"
FORMATS="pdf xml txt csv html"
EXPECTED_pdf=
EXPECTED_xml=
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html=
