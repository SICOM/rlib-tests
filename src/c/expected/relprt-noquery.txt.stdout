Part Page Header
I love bacon
NO DATA

Test Report 2

NO DATA

Test Report 2

NO DATA
Page:   1
Wheeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee:   1
